### PR TITLE
More changes for review 1.1 release 

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/diann_convert.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def cli():
 @click.option("--qvalue_threshold", "-q", type=float)
 @click.pass_context
 def convert(ctx, folder, dia_params, diann_version, charge, missed_cleavages, qvalue_threshold):
-    """This function is designed to convert the DIA-NN output into three standard formats: MSstats, Triqler and mzTab. These documents are
+    """
+    This function is designed to convert the DIA-NN output into three standard formats: MSstats, Triqler and mzTab. These documents are
```
Comment (fabianegli) [inline bin/diann_convert.py:34]:
```suggestion
    Convert DIA-NN output to MSstats, Triqler or mzTab.
    
     The output formats are
```

**Comment by ypriverol** [comment]
@fabianegli when you finish the review, please approve the PR.

File: bin/diann_convert.py [hunk 2]
```diff
@@ -885,14 +900,14 @@ def find_modification(peptide):
 
 
 def calculate_mz(seq, charge):
-    """Remove unknown aminoacids and calculate mz
+    """
+    Calculate the precursor m/z based on the peptide sequence and charge state.
 
-    :param seq: Sequences of peptides
+    :param seq: Sequence peptide
```
Comment (fabianegli) [inline bin/diann_convert.py:908]:
```suggestion
    :param seq: Peptide sequence
```

File: bin/diann_convert.py [hunk 3]
```diff
@@ -30,7 +30,10 @@ def cli():
 @click.option("--qvalue_threshold", "-q", type=float)
 @click.pass_context
 def convert(ctx, folder, dia_params, diann_version, charge, missed_cleavages, qvalue_threshold):
-    """This function is designed to convert the DIA-NN output into three standard formats: MSstats, Triqler and mzTab. These documents are
+    """
+    Convert DIA-NN output to MSstats, Triqler or mzTab.
+
```
Comment (fabianegli) [inline bin/diann_convert.py:35]: Remove trailing whitespace
```suggestion

```

**Review by fabianegli — APPROVED** [review]

**Pull request merged**